### ISSUE TITLE
web-in-60 dynamic dispatch doc refers to an invalid method

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5920]
|<img alt="ltaylor.volks's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ltaylor.volks reported|
|-|-|
|Trac ID|trac#5920|
|Type|defect|
|Created|2012-08-26 17:00:16Z|

Example refers to ``` Calendar.getPage ``` when it should read ``` Calendar.getChild ```

http://twistedmatrix.com/documents/current/web/howto/web-in-60/dynamic-dispatch.html

Attachments:

* [5920-web60docfix.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/6db/6dba4a50ba6ead19f27db265104b6afd1c13e69c/585b2f19ceaaf006e22c948306880739d965bba4.diff) (585 bytes) - added by ltaylor.volks on 2012-08-26 17:03:23Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5920 5920
type__defect defect
reporter__ltaylor_volks ltaylor.volks
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ltaylor_volks ltaylor.volks
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation_easy documentation easy
time__1346000416000000 1346000416000000
changetime__1347137957000000 1347137957000000
version__None None
owner__thijs thijs
cc__jknight cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [35557]) Apply 5920-web60docfix.diff.

Author: ltaylor.volks
Reviewer: thijs
Fixes: [#5920](https://github.com/twisted/twisted/issues/5920)

Correct reference to Calendar.getChild in dynamic dispatch part of the web in 60 seconds tutorial.
